tests/assembly/cmse.rs

```diff
@@ -0,0 +1,26 @@
+//@ assembly-output: emit-asm
+//@ compile-flags: --target thumbv8m.main-none-eabi --crate-type lib -Copt-level=1
+//@ needs-llvm-components: arm
+#![crate_type = "lib"]
+#![feature(abi_c_cmse_nonsecure_call, cmse_nonsecure_entry, no_core, lang_items)]
+#![no_core]
+#[lang = "sized"]
+pub trait Sized {}
+#[lang = "copy"]
+pub trait Copy {}
+
+// CHECK-LABEL: __acle_se_entry_point
+// CHECK: bxns
```

Comment:
This tests end-to-end if we communicated the attribute to LLVM. From the tracking issue ( #75835 ):
> * add a special symbol on the function which is the __acle_se_ prefix and the standard function name
> * constrain the number of parameters to avoid using the Non-Secure stack
> * before returning from the function, clear registers that might contain Secure information
> * use the BXNS instruction to return

So here we check if the label with the `__acle_se_` prefix exists. And that the `bxns` (Branch and Exchange Non-secure) instruction is used to return.